language-server/dm/delegationManager.ml

```diff
@@ -28,13 +27,13 @@ type execution_status =
   | Success of Vernacstate.t option
   | Error of string Loc.located * Vernacstate.t option (* State to use for resiliency *)
 
+(* **alert** calling log from write value causes a loop, since log (from the woker)
+    writes the value to a channel *)
 let write_value { write_to; _ } x =
```

Comment:
```suggestion
let write_value { write_to; _ } x =
(* **alert** calling log from write value causes a loop, since log (from the woker)
    writes the value to a channel *)
let log _ = () in
```